src/main/kotlin/lib/graphics/turtle/awt/CanvasTurtle.kt

```diff
@@ -0,0 +1,72 @@
+package lib.graphics.turtle.awt
+
+import lib.graphics.turtle.Turtle
+import lib.graphics.turtle.TurtleState
+import java.awt.Canvas
+import java.awt.Dimension
+import java.awt.EventQueue.invokeAndWait
+import java.awt.Graphics
+import java.awt.image.BufferedImage
+import java.time.Duration
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+/**
+ * AWT component with [Turtle] interface and buffering
+ * (i. e. what's drawn by turtle is not lost upon GUI refresh)
+ */
+class CanvasTurtle(size: Dimension): Canvas() {
+    private val buffer = BufferedImage(size.width, size.height, BufferedImage.TYPE_INT_RGB)
```

Comment:
```suggestion
    private val buffer = BufferedImage(size.width, size.height, BufferedImage.TYPE_INT_RGB).apply {
        graphics.fillRect(0, 0, size.width, size.height)
    }
```
И будет тебе белый фон

Reply:
Поправил. @gumanoid глянешь?